client/modules/IDE/pages/IDEView.jsx

```diff
@@ -173,6 +173,14 @@ class IDEView extends React.Component {
       // don't warn
       this.props.persistState();
       window.onbeforeunload = null;
+    } else if (route && (/[\s\S]*\/sketches\/?$/.test(route.pathname))) {
```

Comment:
why `[\s\S]`? since it's a url, wouldn't there never be whitespace characters?

Reply:
ok i will remove \s
convert it to
```
/[\S]*\/sketches\/?$/
```